website/index.ejs

```diff
@@ -441,9 +429,15 @@
         </div>
 
         <div class="extension">
-          <%- banner('utilities') %>
-          <h3>Utilities</h3>
-          <p>A bunch of interesting blocks. Originally created by <a href="https://scratch.mit.edu/users/Sheep_maker/">Sheep_maker</a>.</p>
+          <%- banner('clonesplus') %>
```

Comment:
banner('LukeManiaStudios/ClonesPlus')

Reply:
oops 💀

Reply:
I'll get used to this eventually 😭

Reply:
eventually we will clean up the mess that is index.ejs. a mistake like this really should not be able to have "All checks have passed"